include/swift/Driver/DependencyGraph.h

```diff
@@ -268,10 +268,10 @@ class DependencyGraph : public DependencyGraphImpl {
   /// If you want to see how each node gets added to \p visited, pass a local
   /// MarkTracer instance to \p tracer.
   ///
-  /// Assumes that the starting node is "cascading"; that all dependencies must
-  /// be dirty if the start is dirty. Therefore, mark the start. For each
-  /// visited node, add it to visited, and mark it if it cascades. The start
-  /// node is NOT added to visited.
+  /// Assumes that some edge into the starting node is "cascading."
```

Comment:
I like this better, but since no such edge has to actually exist maybe there's still another way to phrase it?

Reply:
Good point! I've reworded.